administrator/language/en-GB/en-GB.com_content.ini

```
@@ -34,6 +34,7 @@ COM_CONTENT_CREATE_ARTICLE_CATEGORY_LABEL="Default Category"
 COM_CONTENT_CREATE_ARTICLE_CATEGORY_DESC="If set to 'Yes', this page will only let you create articles in the category selected below."
 COM_CONTENT_CREATE_ARTICLE_CUSTOM_CANCEL_REDIRECT_DESC="If set to 'Yes', you can set a redirection page, distinct from above 'Submission/Cancel Redirect', to redirect to when user Cancels article submission.<br />If set to 'No', when user Cancels article submission, the user is redirected to the above 'Submission/Cancel Redirect' page."
 COM_CONTENT_CREATE_ARTICLE_CUSTOM_CANCEL_REDIRECT_LABEL="Custom Redirect on Cancel"
+COM_CONTENT_CREATE_ARTICLE_ERROR="When default category is enabled, a category should be selected."
```

```suggestion
COM_CONTENT_CREATE_ARTICLE_ERROR="When default category is enabled, a category has to be selected."
```
"Should" is just a kind of recommendation, not a "must" or a "has to be" or a "needs to".

---

agree. merging now